.github/workflows/test-query-engine-driver-adapters.yml

```diff
@@ -1,57 +1,58 @@
-name: Test Driver Adapters
+name: "QE: driver-adapter integration tests"
 on:
   push:
     branches:
       - main
   pull_request:
     paths-ignore:
-      - '.github/**'
-      - '!.github/workflows/query-engine-driver-adapters.yml'
-      - '.buildkite/**'
-      - '*.md'
-      - 'LICENSE'
-      - 'CODEOWNERS'
-      - 'renovate.json'
+      - "!.github/workflows/test-query-engine-driver-adapters.yml"
+      - ".github/**"
+      - ".buildkite/**"
+      - "*.md"
+      - "LICENSE"
+      - "CODEOWNERS"
+      - "renovate.json"
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   rust-query-engine-tests:
-    name: '${{ matrix.adapter.name }} on node v${{ matrix.node_version }}'
+    name: "${{ matrix.adapter.name }} ${{ matrix.partition }}"
```

Comment:
Suggestion: Either leave node version in output here, or remove it from matrix and hardcode in setup node step.